scripts/tests.sh

```diff
@@ -61,6 +61,6 @@ echo
 # Run tests, sequentially because the API is shared resource
 echo "Running tests ..."
 richgo clean -testcache
-RICHGO_FORCE_COLOR=1 richgo test -p 1 -timeout 360s -v -race -coverprofile=./target/profile.out ./src/... $@
+RICHGO_FORCE_COLOR=1 richgo test -p 1 -timeout 360s -v -race -coverprofile=/tmp/profile.out ./src/... $@
```

Comment:
`goreleaser` vymaze aj `.gitkeep`, ... neda sa to asi nejak vypnut, ... a nechcem ho pred kazdym commitom vytvarat + ... vo funkcionalnych testoch sa vola `goreleaser`, tak `profile.out` nemozem ukladat do `target`, ktory sa premaze.

Preto ho ulozim do `tmp`, zatial sa nikde inde nepouziva.

Reply:
a nieje to kvoli tomu ze `.gitkeep` je zmazeny v z tohto repa uplne? Ked sa repo checkoutne tak sa nevytvori target zlozka.

Reply:
jj, presne tak ... ako pisem, ze to by som musel vytvarat `.gitkeep` pri kazdom commite, aby tam bol.
Preto je jednoduchsie, nech si zlozku `target` vytvori sam `goreleaser` a nebude v gite.

Reply:
jasne, ja som mal zato ze ten .gitkeep tam bude furt a jeho pripadne zmazanie sa necommitne, ale je to zbytocna praca naviac, takze takto to je lepsie vlastne.